Wikipedia/assets/index.html

```diff
@@ -6,7 +6,11 @@
     <meta name="viewport" id="viewport" content="width=device-width, user-scalable=no">
     <link href="styles.css" rel="stylesheet" type="text/css"></link>
     <link href="styleoverrides.css" rel="stylesheet" type="text/css"></link>
-    
+    <style type="text/css">
```

Comment:
I'd like to revisit having the text size set via strings substitution when we have more time to visit how transforms are handled. But in the mean time this is fine with one exception:

- as things are presently structured the editing interface's `preview.html` is processed by the same `stringWithFormat`, so for it to continue working  `preview.html` will need `%@` to be added to it as well. It can just be surrounded by html comments - something like: `<!-- %@ (needed for consistency with index.html) -->`. Will also have to come before the `%@` already in `preview.html`.